databases/week2/part1.sql

```diff
@@ -0,0 +1,60 @@
+/*  Part 1: Working with tasks
+ Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+Change the title of a task
+Change a task due date
+Change a task status
+Mark a task as complete
+Delete a task
+ */
+USE DATABASE `hyfmodels`;
+
+select *
+from user
+where name like 'Anamika%';
+
+select *
+from status;
+
+select *
+from task;
+
+select * from user_task where user_id = 12;
+select * from user_task where task_id = 1;
```

Comment:
This testing part can be removed as they are not  part of the hw

Reply:
I don't know if you have reviewed each commit. This has been already deleted